src/routes/tradesToPublish.ts

```diff
@@ -10,26 +10,59 @@ const logger = getLogger();
 
 const previousTradesLookup: Map<string, Trade> = new Map<string, Trade>();
 
+export interface LedgerToPublish {
+  name: string;
+  network: string;
+}
+
+export interface RateToPublish {
```

Comment:
🔧 Given, that this was named `trade` in most of the other parts now we should probably name it trade instead of `Rate`. (`Trade` is not valid any more as well, should be `Offer` I think, but that can be done in a follow up PR).

Reply:
Hum, ok, let's do an Offer PR next week then :)

Reply:
PR done :)